src/Task/PhpStan.php

```diff
@@ -56,7 +60,15 @@ public function canRunInContext(ContextInterface $context)
     public function run(ContextInterface $context)
     {
         $config = $this->getConfiguration();
-        $files = $context->getFiles()->extensions($config['triggered_by']);
+        $files = $context
+            ->getFiles()
+            ->notPaths($config['ignore_patterns'])
+            ->extensions($config['triggered_by']);
+
+        $forcedFiles = $context->getFiles()->paths($config['force_patterns']);
+        foreach ($forcedFiles as $forcedFile) {
+            $files->add($forcedFile);
```

Comment:
Shouldn't we do some intersection here to make sure the files aren't added twice?